Src/library/base/elks/kernel/file_name.e

```diff
@@ -9,6 +9,8 @@ note
 class
 	FILE_NAME
 
+obsolete "This class is obsolete, use PATH instead"
```

Comment:
It is missing a date ... I guess the compiler reported this missing date as warning.

Reply:
fixed

Reply:
added date